src/prometheus_metric.erl

```diff
@@ -75,7 +75,7 @@ as well as handling metric labels and data.
         registry => prometheus_registry:registry(),
         constant_labels => [{atom(), term()}],
         labels => labels(),
-        data => any(),
+        data => term(),
```

Comment:
If we wanna be consistent, we can introduce `rebar3_lint` rule `consistent_generic_type`. I'm Ok to introduce `rebar3_lint` with a very limited ruleset, if you want... but that's another issue. I just noticed `any()` was being changed to `term()` and that triggered this thought in me.

Reply:
Didn't know that rule, going to have a look at it 👌🏽

Reply:
The rule actually proposes what you did, and is proposed by Erlang/OTP: `term()` over `any()`, since more idiomatic.